src/app/retrospective/retrospective-feedback/retrospective-feedback.component.ts

```diff
@@ -128,7 +130,9 @@ export class RetrospectiveFeedbackComponent implements OnInit, OnChanges, OnDest
             stopEditingWhenGridLosesFocus: true,
             suppressDragLeaveHidesColumns: true,
             suppressScrollOnNewData: true,
-            onColumnVisible: (event) => this.gridApi.sizeColumnsToFit()
+            onColumnVisible: (event) => this.gridApi.sizeColumnsToFit(),
+            onFilterChanged: (event) =>{this.filterService.setFilterData(this.feedbackSubType, this.gridApi.getFilterModel());
```

Comment:
Please use this if the statement doesn't exceed the defined line length.
```suggestion
            onFilterChanged: (event) => this.filterService.setFilterData(this.feedbackSubType, this.gridApi.getFilterModel())
```

Reply:
done